src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,14 +1,28 @@
 import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as filterActions } from '../../features/filter';
+import { Status } from '../../types/Status';
 
 export const TodoFilter: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const query = useAppSelector(state => state.filter.query);
+
+  const onChangeStatus = (event: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
```suggestion
  const handleStatusChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364